src/strucclustutils/structureconvertalis.cpp

```diff
@@ -522,6 +525,14 @@ int structureconvertalis(int argc, const char **argv, const Command &command) {
                     tmres = tmaligner->computeTMscore(targetCaData, &targetCaData[res.dbLen], &targetCaData[res.dbLen+res.dbLen], res.dbLen,
                                                       res.qStartPos, res.dbStartPos, Matcher::uncompressAlignment(res.backtrace));
                 }
+                LDDTcalculator::LDDTscoreResult lddtres;
+                if(needLDDT) {
+                    lddtcalculator = new LDDTcalculator(res.qLen, res.dbLen);
```

Comment:
This is a problem. We should reuse the LDDTcalculator object to not always allocate new memory.